charts/atlantis/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v0.20.1
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.5.1
+version: 4.6.0
```

Comment:
Before merging, we'll have to merge the default branch here and then update this line for the next minor version.